motor/providers/ssh/session.go

```diff
@@ -201,14 +204,48 @@ func prepareConnection(pCfg *providers.Config) ([]ssh.AuthMethod, []io.Closer, e
 		}
 	}
 
-	if len(sshSigners) > 0 {
```

Comment:
I moved this below loading the ssh-agent certs since anything that is appended to `sshSigners` after that is simply ignored. I feel like this was a bug.

Reply:
This should be correct here. The ssh agents authentication is activated in the cli if the user has not provided anything. Therefore we can remove the fallback to ssh-agent here.

Reply:
so this code is not needed at all then?